snapshot/overlay/overlay.go

```diff
@@ -174,7 +185,7 @@ func (o *Snapshotter) Walk(fn func(snapshot.Info) error) error {
 	})
 }
 
-func (o *Snapshotter) newActiveDir(key string) (*activeDir, error) {
+func (o *Snapshotter) newActiveDir(key string, readonly bool) (*activeDir, error) {
```

Comment:
This extra argument is only used in two places and is hard-coded in each. Can this logic be handled in the caller instead of adding a bool arg?